cookbooks/aws-parallelcluster-test/metadata.rb

```diff
@@ -14,3 +14,4 @@
 supports 'ubuntu', '>= 18.04'
 
 depends 'aws-parallelcluster-common', '~> 3.6.0'
+depends 'aws-parallelcluster-install', '~> 3.6.0'
```

Comment:
Why are we adding a dependency for -install cookbook?

Reply:
It was needed to test the change in the mysql-validate. Probably it is not needed anymore